app/src/main/java/org/simple/clinic/patient/PatientAddress.kt

```diff
@@ -36,6 +36,17 @@ data class PatientAddress(
 
     val deletedAt: Instant?
 ) : Parcelable {
+
+  val addressFieldsString: String
```

Comment:
Can we rename this to something like `completeAddress` and drop the `String` suffix since that is already in the type?